server/lib/express/routes/api/v1/users/[user_id]/createSvg.js

```diff
@@ -401,7 +401,7 @@ async function createSvg(userData, options = {}) {
                 font-weight: 600;
                 color: ${variables.colors.text.primary};
               ">
-                No Activity
+                ${options.noActivityMessagePrimary || 'No Activity'}
```

Comment:
Add it to the defaultOptions object in `server/lib/express/routes/api/v1/users/[user_id]/index.js` file